android/src/main/java/com/brentvatne/exoplayer/AspectRatioFrameLayout.kt

```diff
@@ -21,50 +21,27 @@ class AspectRatioFrameLayout(context: Context) : FrameLayout(context) {
     companion object {
         private const val MAX_ASPECT_RATIO_DEFORMATION_FRACTION = 0.01f
     }
-    private var videoAspectRatio: Float = 0f
-    private var resizeMode: Int = ResizeMode.RESIZE_MODE_FIT
 
-    /**
-     * Set the aspect ratio that this view should satisfy.
-     *
-     * @param widthHeightRatio The width to height ratio.
-     */
-    fun setAspectRatio(widthHeightRatio: Float) {
-        if (videoAspectRatio != widthHeightRatio) {
-            videoAspectRatio = widthHeightRatio
-            requestLayout()
+    var videoAspectRatio: Float = 0f
+        set(value) {
+            if (value != videoAspectRatio) {
```

Comment:
nit;
I recommend using `field` instead of `videoAspectRatio` here because having a custom getter can lead to side effects, But it is not currently an issue.

Reply:
@YangJonghun  I agree with you about the side effects. I have applied your suggestion.